src/main/java/com/example/Restaurant_Management_System_REST_API/service/CustomerService.java

```diff
@@ -65,11 +70,20 @@ public CustomerDTOResponse create(CustomerDTORequest customerDTORequest) {
                     }
                 })
                 .collect(Collectors.toSet());
+    }
 
-        customer.setAuthorities(authorities);
-        customerRepository.save(customer);
+    private static void checkIfAuthoritiesAreNull(CustomerDTORequest customerDTORequest) {
```

Comment:
Is there a reason behind `static` here? sorry if I can't see obvious here :)

Reply:
the reason was I didnt notice while it was extracted. Static is not needed according to me